src/SUMMARY.md

```diff
@@ -30,7 +30,8 @@
   - [Export to Web](toolchain/export-web.md)
 - [Recipes](recipes/index.md)
   - [Custom resources](recipes/custom-resources.md)
-  - [Editor plugins](recipes/editor-plugin.md)
+  - [Editor plugins](recipes/editor-plugin/index.md)
+    - [Inspector plugins](recipes/editor-plugin/inspector-plugins.md)
```

Comment:
I would probably keep those in one page, as they are closely related and the current page is very short. Just use a h2 for the inspector plugins.

Then, you also wouldn't need a new folder. The images could be named `inspector-before/after.png`.

Reply:
I thought it could be extended with others exmaples 🤔

Reply:
Ah you mean [others from this list](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/index.html)? Good point, then maybe leave it.

Can you add a HTML comment (which isn't rendered) on the main editor plugin site, like
```
<!-- TODO: more plugins from https://docs.godotengine.org/en/stable/tutorials/plugins/editor/index.html -->
```
?

Reply:
yep, have a look